src/buf.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+version: v1
+breaking:
+  use:
+    - FILE
+lint:
+  use:
+    - DEFAULT
+build:
+  excludes:
```

Comment:
A comment would be nice as to why those files are excluded -- I believe the original ticket does list the reason , so just copy it over as a comment here if you do not mind.

Reply:
✅ Done.